docs/markdown_docs/plugins/jspsych-iat.md

```diff
@@ -0,0 +1,227 @@
+# jspsych-iat plugin
+
+This plugin displays an image or HTML-formatted content that is the stimulus of the experiement. Each stimulus is assigned a category and a key that the user presses to give a response. The stimulus can be displayed until a response is given, or for a pre-determined amount of time. If the stimulus is displayed until a response is given, then an image to indicate if the user pressed the wrong key can be displayed. The trial can be ended automatically if the subject has failed to respond within a fixed length of time.
```

Comment:
This description should be updated to convey how the IAT works (or provide a link to a reference on the IAT, perhaps)